### ISSUE TITLE
need a fix in buy me coffee

### ISSUE THREAD

**Issue by sandeep0009** [Vikash-8090-Yadav/SaL--dApp#138]
<img width="617" alt="image" src="https://github.com/Vikash-8090-Yadav/SaL--dApp/assets/63727946/078f1db8-aeb6-4f5d-833b-4a6e12ef49a1">

here the button looks pretty ugly and also we dont require expandable textarea as issues should be very up to the point rather telling whole story

**Comment by payalpm** [comment]
I am interested in working on this issue @sandeep0009 assign me plz little bit explore this project

**Comment by roshangeorge97** [comment]
> <img alt="image" width="617" src="https://user-images.githubusercontent.com/63727946/241355889-078f1db8-aeb6-4f5d-833b-4a6e12ef49a1.png">
> here the button looks pretty ugly and also we dont require expandable textarea as issues should be very up to the point rather telling whole story

Yeah it would be a good UI Issue to work with, I am assigning you with this issue
